pages/linux/xmount.md

```diff
@@ -0,0 +1,17 @@
+# xmount
+
+> Allows you to convert on-the-fly between multiple input and output harddisk image types with optional write cache support.
+> Creates a virtual file system using FUSE (Filesystem in Userspace) that contains a virtual representation of the input image.
+> More information: <https://www.pinguin.lu/xmount>.
+
+- Mount an RAW-Imagefile into a DMG-container file:
+
+`xmount --in raw IMAGE.dd --out dmg MOUNTPOINT`
```

Comment:
I'd rearrange the command this way:
```suggestion
- Mount a RAW image file into a DMG container file:

`xmount --in {{raw}} --out {{dmg}} {{path/to/image.dd}} {{mountpoint}}`
```
Because having `raw` right before the `.dd` file, and `dmg` right before `mointpoint` makes seem as though the `.dd` is part of `--in` and `mountpoint` is part of `--out`, when in reality they the files have nothing to do with the flags — the flags merely indicate the input/output file _types_.

Reply:
I like the suggestion with the {{ bracklets but we have to change the order because the description of the format is part  of the syntax --in or --out.

Your syntax would trigger this warning on execution:
_WARNING: ParseCmdLine@711 : You are using a deprecated --in option syntax which will be removed in the next release. Please see the man page on how to use the new syntax._
```suggestion
- Mount a RAW image file into a DMG container file:

`xmount --in {{raw}} {{path/to/image.dd}} --out {{dmg}} {{mountpoint}}`
```